src/explore-education-statistics-common/src/services/publicationService.ts

```diff
@@ -93,13 +93,19 @@ export interface KeyStatisticBase {
   updated?: string;
 }
 
+// eslint-disable-next-line no-shadow
```

Comment:
On an aside, I've added this no-shadow disable, copying a different place that we used an enum in the codebase, but I think we can change the eslint config so these aren't necessary:
https://typescript-eslint.io/rules/no-shadow/
```
module.exports = {
  "rules": {
    // Note: you must disable the base rule as it can report incorrect errors
    "no-shadow": "off",
    "@typescript-eslint/no-shadow": "warn"
  }
};
```
Not sure if this is right or not, so I've followed the established way.

Reply:
Talked to Nick - going to add the config so we can remove the disable lines